src/verso-manual/VersoManual/Bibliography.lean

```diff
@@ -77,6 +88,7 @@ inductive Citable where
   | thesis : Thesis → Citable
   | arXiv : ArXiv → Citable
   | article : Article → Citable
+  | book : Book → Citable
```

Comment:
This is fine, but I do feel like this datatype is getting out of hand and it will soon be time to find an open representation here rather than the current one. A type class, perhaps.

Reply:
(this is not something for you to do, unless you're inspired)